lib/member_page.rb

```diff
@@ -0,0 +1,29 @@
+#!/bin/env ruby
```

Comment:
here, and elsewhere, it's not really clear why you've introduced shebangs. These aren't scripts that are ever going to be run directly.